py-rse/provenance.Rmd

```diff
@@ -276,13 +72,16 @@ For example,
 NASA has published
 the [Goddard Institute for Space Studies Surface Temperature Analysis][gistemp],
 which estimates the global average surface temperature
-based on land and ocean weather observations,
+based on thousands of land and ocean weather observations,
 because a simple metric of global warming is expensive to produce
 and is useful in many research contexts.
 
 If a report involves a new dataset,
 such as observations collected during a field experiment,
-then they need to be published following the FAIR Principles.
+then those data need to be published in their raw (unprocessed) form.
```

Comment:
I think we use "data" as singular elsewhere, so this would be "then that data needs to be published in its raw..."